cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -145,240 +145,240 @@ spec:
     spec:
       serviceAccountName: rook-ceph-system
       containers:
-      - name: rook-ceph-operator
```

Comment:
Please revert the whitespace changes so the diff will be limited to the intended changes. Are there any intended changes in this yaml?

Reply:
no, my bad that I didn't made changes to the yaml after testing it.

Reply:
My vscode is adding extra quotes, I have to look into it.

Reply:
Done!, please review now, I have fixed all this.
thanks